contracts/PrepaidCardManager.sol

```diff
@@ -142,6 +144,7 @@ contract PrepaidCardManager is Ownable, Versionable, Safe {
     uint256 _minAmount,
     uint256 _maxAmount,
     address[] calldata _contractSigners,
+    address[] calldata _trustedCallersForCreatingPrepaidCardsWithIssuer,
```

Comment:
make sure to update the configuration scripts to set this: https://github.com/cardstack/card-pay-protocol/blob/main/scripts/deploy/config/default/PrepaidCardManager.ts#L22-L59